controllers/yawol-controller/loadbalancerset/loadbalancerset_controller_test.go

```diff
@@ -382,7 +382,29 @@ func TestIsMachineReady(t *testing.T) {
 func TestShouldMachineBeDeleted(t *testing.T) {
 	t.Run("Do not delete if there are no conditions shortly after creation", func(t *testing.T) {
 		machine := &yawolv1beta1.LoadBalancerMachine{
-			ObjectMeta: metav1.ObjectMeta{CreationTimestamp: metav1.Time{Time: time.Now().Add(-9 * time.Minute)}},
+			ObjectMeta: metav1.ObjectMeta{
+				CreationTimestamp: metav1.Time{Time: time.Now().Add(-9 * time.Minute)},
+			},
+			Status: yawolv1beta1.LoadBalancerMachineStatus{
+				Conditions: nil,
+			},
+		}
+		got, _ := shouldMachineBeDeleted(machine)
+		want := false
+
+		if !reflect.DeepEqual(got, want) {
```

Comment:
why do we use  `reflect.DeepEqual` for this? AFAIK it uses go's`==` internally for primitive types anyway.

Reply:
You are right, I'm just copy pasting to look like the other tests for `shouldMachineBeDeleted`

Reply:
okay :D. Yeah doesn't really matter. I will resolve this